generators/entity-server/templates/ui/widgets/tableWidget/src/components/__types__/entity.js.ejs

```diff
@@ -0,0 +1,28 @@
+<%# { "useBluePrint": true, "renameTo": "`/${generator.entityInstance}/tableWidget/src/components/__types__/${generator.entityInstance}.js`" } -%>
+import PropTypes from 'prop-types';
+
+const <%= entityInstance %>Type = PropTypes.arrayOf(
+    PropTypes.shape({
+        id: PropTypes.number,
+        <%_
+        function mapTypeToPropType (field) {
```

Comment:
Can we extract this out to a utility so we can use it across the different widgets or is it specific to what is happening in the table?

Reply:
I was planning to - part of it is generic and should be modulized.